src/System.Net.Security/src/System/Net/SecureProtocols/NegoState.cs

```diff
@@ -838,20 +776,26 @@ private static void ReadCallback(IAsyncResult transportResult)
             }
         }
 
-        private unsafe byte[] GetOutgoingBlob(byte[] incomingBlob, ref Win32Exception e)
+        internal static bool IsError(SecurityStatusPal status)
         {
-            Interop.SecurityStatus statusCode;
+            return ((int)status.ErrorCode >= (int)SecurityStatusPalErrorCode.OutOfMemory);
+        }
+
+        private unsafe byte[] GetOutgoingBlob(byte[] incomingBlob, ref Exception e)
```

Comment:
I presume that now there is no reason for this type change here? The PAL will always return a Win32Exception on all platforms.
We should revert the function signature.